manifests/install.pp

```diff
@@ -187,7 +208,7 @@
     timeout => 1800,
   }
   # Install gitlab with the appropriate package manager (rpm or dpkg)
-  package { 'gitlab':
+  package { "$::gitlab::install::gitlab_pkg":
```

Comment:
@ahuffman please look up what [name in package](https://docs.puppetlabs.com/references/3.stable/type.html#package-attribute-name) (or for that matter, [most [defined] types](https://docs.puppetlabs.com/puppet/latest/reference/lang_resources.html#namenamevar) does).